libs/ledger-live-common/src/generated/specs.ts

```diff
@@ -44,4 +43,5 @@ export default {
   tron,
   xrp,
   icon,
+  elrond,
```

Comment:
I think the original sorting was alphabetical.

Reply:
I don't think it was.